src/locale/nb.js

```diff
@@ -45,6 +45,8 @@ export default moment.defineLocale('nb', {
         hh: '%d timer',
         d: 'en dag',
         dd: '%d dager',
+        w: 'en uke',
+        ww: '%d uker',
```

Comment:
Looks good! If you want to add nynorsk (nn) as well, it is 'ei veke' / '%d veker'

Reply:
Thanks @sigurdga ! nn added !